<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Teacher</class>
 <widget class="QMainWindow" name="Teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Teacher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">#tabWidget {

background:rgb(233, 185, 110);}</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="styleSheet">
        <string notr="true">#tab {

background:rgb(233, 185, 110);}</string>
       </property>
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>560</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Assign Grade</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Previous">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>650</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Next">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>650</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>650</width>
          <height>600</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
border-style: solid;
border-width: 1px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QStackedWidget" name="Painting_area_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>650</width>
           <height>600</height>
          </rect>
         </property>
         <widget class="QWidget" name="page"/>
        </widget>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>400</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Comment</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="Comment_box">
        <property name="geometry">
         <rect>
          <x>890</x>
          <y>420</y>
          <width>341</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Save_feedback">
        <property name="geometry">
         <rect>
          <x>1150</x>
          <y>580</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#Save_feedback {
background:rgb(52, 101, 164);

}

#Save_feedback:hover{
background:rgb(114, 159, 207);
}</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>80</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Student</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>1060</x>
          <y>80</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Tasks Completed</string>
        </property>
       </widget>
       <widget class="QListWidget" name="graded_students_list">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>110</y>
          <width>151</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="Completed_tasks_list">
        <property name="geometry">
         <rect>
          <x>1060</x>
          <y>110</y>
          <width>161</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Grade">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>580</y>
          <width>62</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="styleSheet">
        <string notr="true">#tab_2{

background:rgb(233, 185, 110);}</string>
       </property>
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>550</width>
          <height>500</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
border-style: solid;
border-width: 1px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QStackedWidget" name="Painting_area">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>550</width>
           <height>500</height>
          </rect>
         </property>
         <widget class="QWidget" name="page_3"/>
        </widget>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>80</y>
          <width>101</width>
          <height>301</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="Assign">
           <property name="styleSheet">
            <string notr="true">#Assign {
background:rgb(52, 101, 164);

}

#Assign:hover{
background:rgb(114, 159, 207);
}</string>
           </property>
           <property name="text">
            <string>Assign &gt;&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Drop">
           <property name="styleSheet">
            <string notr="true">#Drop {
background:rgb(52, 101, 164);

}

#Drop:hover{
background:rgb(114, 159, 207);
}</string>
           </property>
           <property name="text">
            <string>&lt;&lt; Drop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Assign_all">
           <property name="styleSheet">
            <string notr="true">#Assign_all {
background:rgb(52, 101, 164);

}

#Assign_all:hover{
background:rgb(114, 159, 207);
}</string>
           </property>
           <property name="text">
            <string>Assign to all</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="Delete">
           <property name="styleSheet">
            <string notr="true">#Delete {
background:rgb(52, 101, 164);

}

#Delete:hover{
background:rgb(114, 159, 207);
}</string>
           </property>
           <property name="text">
            <string>Delete Task</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QListWidget" name="Task_list">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>80</y>
          <width>171</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>60</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Tasks List</string>
        </property>
       </widget>
       <widget class="QListWidget" name="Student_list">
        <property name="geometry">
         <rect>
          <x>970</x>
          <y>80</y>
          <width>171</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="Assigned_tasks_list">
        <property name="geometry">
         <rect>
          <x>970</x>
          <y>440</y>
          <width>171</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>970</x>
          <y>60</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Students</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>970</x>
          <y>420</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Assigned Tasks</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <property name="styleSheet">
        <string notr="true">#tab_3 {

background:rgb(233, 185, 110);}</string>
       </property>
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>800</width>
          <height>600</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
border-style: solid;
border-width: 1px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QStackedWidget" name="Painting_area_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>800</width>
           <height>600</height>
          </rect>
         </property>
         <widget class="QWidget" name="page_4"/>
        </widget>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>130</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Letter Name:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Letter_name">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>150</y>
          <width>261</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QTextEdit" name="Description">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>250</y>
          <width>261</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>230</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Save">
        <property name="geometry">
         <rect>
          <x>1050</x>
          <y>490</y>
          <width>93</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#Save {
background:rgb(52, 101, 164);

}

#Save:hover{
background:rgb(114, 159, 207);
}</string>
        </property>
        <property name="text">
         <string>Save Task</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Clear">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>490</y>
          <width>93</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#Clear {
background:rgb(52, 101, 164);

}

#Clear:hover{
background:rgb(114, 159, 207);
}</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
